react-game/src/components/TrialRow/TrialRow.js

```diff
@@ -0,0 +1,113 @@
+import { useState, useRef, useEffect } from 'react';
+import CheckButton from '../Buttons/CheckButton';
+import WhiteCircle from '../ResultCircles/WhiteCircle';
+import BlueCircle from '../ResultCircles/BlueCircle';
+import EnteredNumber from './EnteredNumber';
+import classes from './Trial.module.css';
+
+
+function TrialRow({ rowIndex, selectedIndex, setSelectedIndex, secretCode, setGameStatus, clearScreen, setClearScreen }) {
+
+    //STATES & HOOKS------------------------------------------------------------------
+    const [correctNumberWithIndex, setcorrectNumberWithIndex] = useState(0);
```

Comment:
it should be setCorrectNumberWithIndex(C capital letter)

Reply:
done